src/components/Board/index.js

```diff
@@ -216,8 +216,10 @@ function BoardContainer({
     if (!coordinates.source) return
 
     isAColumnMove(event.type)
-      ? onColumnDragEnd({ ...coordinates, subject: board.columns[coordinates.source.fromPosition] })
-      : onCardDragEnd({ ...coordinates, subject: getCard(board, coordinates.source) })
+      ? isValidColumnCoordinates(coordinates) &&
```

Comment:
I think you should define a method by what it does, not by how it does.

A suggestion would be:
```suggestion
      ? isMovingAColumnToAnotherPosition(coordinates) &&
```